uk_bin_collection/uk_bin_collection/councils/LondonBoroughOfRichmondUponThames.py

```
@@ -12,118 +10,90 @@
 
 
 class CouncilClass(AbstractGetBinDataClass):
-    """
-    Richmond upon Thames – parse the static My Property page.
-    No Selenium. No BeautifulSoup. Just requests + regex tailored to the current markup.
-    """
+    """Richmond upon Thames – parse My Richmond property page."""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)